samples/logging/README.rst

```diff
@@ -125,24 +125,40 @@ sample application (i.e., ``samples/logging``) and type:
 
 .. code-block:: console
 
-   $ west build -b nrf52840dk_nrf52840 samples/logging
+   $ west build -b nrf52840dk_nrf52840 samples/logging -p
    $ west flash
 
-nRF9160 Feather
----------------
+nRF9160 Feather or Thingy:91
+----------------------------
 
-On your host computer open a terminal window, locate the source code of this
-sample application (i.e., ``samples/logging``) and type:
+A full nRF9160 firmware `flashing overview`_ is available in the Golioth docs.
+Please use that resource to install the tools necessary to flash these devices.
+
+Configure the build for the MCUboot bootloader by adding this line to the
+configuration file (e.g. ``prj.conf``):
+
+.. code-block:: cfg
+
+   CONFIG_BOOTLOADER_MCUBOOT=y
+
+On your host computer open a terminal window, locate the directory that contains
+the sample folder (i.e., ``~/zephyr-nrf/modules/lib/golioth``) and type:
 
 .. code-block:: console
 
-   $ west build -b circuitdojo_feather_nrf9160_ns samples/logging
+   $ #Build for the Circuit Dojo nrf9160 Feather:
+   $ west build -b circuitdojo_feather_nrf9160_ns samples/lightdb_led -p
+   $ #or build for the Thingy:91:
+   $ west build -b thingy91_nrf9160_ns samples/lightdb_led -p
 
 Enter bootloader and use ``mcumgr`` (or ``newtmgr``) to flash firmware:
 
 .. code-block:: console
 
-   $ mcumgr --conntype serial --connstring /dev/ttyUSB0,baudrate=1000000 build/zephyr/app_update.bin
+   $ #Flashing the Circuit Dojo nRF9160 Feather
+   $ newtmgr --conntype=serial --connstring='dev=/dev/ttyUSB0,baud=1000000' image upload build/zephyr/app_update.bin
+   $ #Flashing example for Thingy:91
+   $ newtmgr --conntype=serial --connstring='dev=/dev/ttyACM0,baud=115200' image upload build/zephyr/app_update.bin
```

Comment:
There should be a reason to use `newtmgr` instead of `mcumgr`. Please put this change in separate commit and make sure to write down the reason of this change. Otherwise we will play ping-pong with switching between `mcumgr` and `newtmgr` if there is no real reason of using one or the other.

Reply:
Okay, will do. When flashing the feather, the version of newtmgr that we recommend in our docs flashes much faster. We've normalized docs to use newtmgr for nRF91 examples.

Reply:
I started changing over the docs to use `mcumgr` instead of `newtmgr`. But then in testing I found that when an upload stalls (fairly frequently), you cannot stop the `mcumgr` process with CTRL-C; it must be killed from another terminal ([there's an open issue](https://github.com/apache/mynewt-mcumgr-cli/issues/21)).

This is a pretty bad user experience. On the one hand, `mcumgr` is an easier install process, but on the other hand `newtmgr` can be stopped with a keyboard interrupt.

Any thoughts on this?